### PR TITLE
fix: fathom analytics

### PR DESCRIPTION
**Pull request by pgray-hiro**
## Description

Fathom tracking was broken for the site, after talking to Thomas I updated the domain it is pointed at and also established a new dashboard on Fathom that represents the correct domain.

## Checklist

- [x] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] New links to files and images were verified
- [x] For fixes/refactors: all existing references were identified and replaced
- [x] [Style guide](https://developers.google.com/style) was reviewed and applied
- [x] Clear code samples were provided
- [x] People were tagged for review

### CODE REVIEW

**Review by agraebe — APPROVED** [review]
🎉

File: next.config.js [hunk 1]
```diff
@@ -901,7 +901,7 @@ module.exports = withFonts(
       trailingSlash: true,
     },
     env: {
-      FATHOM_ID: 'EPNTIXUM',
+      FATHOM_ID: 'FOEMPXUV',
```
Comment (aulneau) [inline next.config.js:904]: why is this change needed? I'd be worried about losing historic data for the docs

Comment (pgray-hiro) [reply next.config.js:904]: We already have a significant gap in the historical analytics due to this breakage.  Since you can't rename a site on Fathom I thought it would be good to have a dashboard with the appropriate `docs.stacks.co` name, no reason we can't keep the old dashboard around for historical reasons though.

Comment (aulneau) [reply next.config.js:904]: ah makes sense

**Review by aulneau — APPROVED** [review]
